docs-ref-conceptual/manage-azure-subscriptions-azure-cli.md

```diff
@@ -1,58 +1,161 @@
 
-```azurecli-interactive
+To run only a single command with a different subscription, use the **--subscription** argument:
+
+```azurecli
 az vm create --subscription "My Demos" --resource-group MyGroup --name NewVM --image Ubuntu
-```
\ No newline at end of file
+```
+
+## Azure CLI config file
+
+There's a configuration file, named *config,* in this location on the computer where you run Azure CLI:
+
+- *$HOME/.azure* on Linux and macOS
+- *%USERPROFILE%\.azure* on Windows
+
+In general, don't edit this file directly. Instead, use the [az config](/cli/azure/config) commands.
+
+Try setting your default location by using the [az config set](/cli/azure/config#az_config_set) command:
+
+```azurecli
+az config set defaults.location=eastus
+```
+
+> [!TIP]
+> To see all available locations for the current subscription, use the [az account list-locations](/cli/azure/account#az_account_list_locations) command.
+
+## Set Azure CLI configuration values
+
+To see your current configuration, run the [az config get](/cli/azure/config#az_config_get) command:
+
+```azurecli
+az config get
+```
+
+This output shows the default location that you set:
+
+```output
+  "defaults": [
+    {
+      "name": "location",
+      "source": "%USERPROFILE%\\.azure\\config",
+      "value": "eastus"
+    }
+```
+
+To set your default subscription for storage commands:
+
+```azurecli
+az config set storage.account="My Demos"
+```
+
+The command sets a value for a specified key, in this case `storage.account`. For available configuration options, see [Azure CLI configuration](/cli/azure/azure-cli-configuration).
+
+Run [az config get](/cli/azure/config#az_config_get) again to see the value for `storage.account`:
+
+```output
+  "storage": [
+    {
+      "name": "account",
+      "source": "%USERPROFILE%\\.azure\\config",
+      "value": "My Demos"
+    }
+```
+
+You can change an existing value by using the [az config set](/cli/azure/config#az_config_set) command again, or remove the value by running [az config unset](/cli/azure/config#az_config_unset):
+
+```azurecli
+az config unset storage.account
+```
+
+The value for `storage` is now empty:
+
+```output
+  "storage": []
+```
+
+## Access tokens
+
+Azure CLI creates an access token when you sign in by using [az login](/cli/azure/reference-index#az_login). This access token is stored in the *accessTokens.json* file in the *.azure* directory.
+
+> [!CAUTION]
+> The *accessTokens.json* file and other files in this directory contain unencrypted values. Do not share the contents of this directory.
+
+The access token expires after, at most, an hour. Azure CLI uses a second token, called a refresh token, to get a new token when needed. The refresh token is also stored in the *accessTokens.json* file. The server manages this token.
+
+Unless you sign out by using [az logout](/cli/azure/reference-index#az_logout), you can continue to run commands without signing in again.
+
```

Comment:
This article needs information on az config param-persist which also can set a default subscription value.

How does the Azure CLI cache interact with stored subscription information?

When does the az account set value expire?

How do I lock a subscription?

Please expand on as many of these commands as possible --> https://docs.microsoft.com/en-us/cli/azure/account